docs/releases/3.2.txt

```diff
@@ -345,6 +345,12 @@ Validators
 * The :class:`.ValidationError` equality operator now ignores ``messages`` and
   ``params`` ordering.
 
+Miscellaneous
+~~~~~~~~~~~~~
+
+* :meth:`django.core.paginator.Paginator.get_elided_page_range` has been
```

Comment:
Maybe:
```
The new :meth:`django.core.paginator.Paginator.get_elided_page_range` allows
customizing ... on ``Paginator`` subclasses ...
```

Reply:
I don't think we really need to mention subclasses here. Ultimately I expect this to be flexible enough for nearly all use cases as the values of `on_ends` and `on_each_side` can be tweaked to handle almost every combination.